tests/MongoDBDriverManager_debugInfo.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+MongoDB\Driver\Manager::__debugInfo()
+--FILE--
+<?php
+$manager = new MongoDB\Driver\Manager();
+
+$command = new MongoDB\Driver\Command(array('ping' => 1));
+$manager->executeCommand("test", $command);
+
+var_dump( $manager->__debugInfo() );
+?>
+--EXPECTF--
+array(2) {
+  ["uri"]=>
+  string(%d) "%s"
+  ["cluster"]=>
+  array(1) {
+    [0]=>
+    array(10) {
+      ["host"]=>
+      string(%d) "%s"
+      ["port"]=>
+      int(%d)
+      ["type"]=>
+      int(1)
+      ["is_primary"]=>
+      bool(false)
+      ["is_secondary"]=>
+      bool(false)
+      ["is_arbiter"]=>
+      bool(false)
+      ["is_hidden"]=>
+      bool(false)
+      ["is_passive"]=>
+      bool(false)
+      ["last_is_master"]=>
+      array(8) {
+        ["ismaster"]=>
+        bool(true)
+        ["maxBsonObjectSize"]=>
+        int(%d)
+        ["maxMessageSizeBytes"]=>
+        int(%d)
+        ["maxWriteBatchSize"]=>
+        int(%d)
+        ["localTime"]=>
+        object(MongoDB\BSON\UTCDateTime)#4 (1) {
+          ["milliseconds"]=>
+          int(%d)
+        }
+        ["maxWireVersion"]=>
+        int(%d)
+        ["minWireVersion"]=>
+        int(0)
```

Comment:
I think this can be `%d` as well, in case future versions bump their minimum wire version.